port-forward-fun.conf

```diff
@@ -1,19 +0,0 @@
-##################### Notes ########################
-#                                                  #
-#  So, you wanna run a single Node server, huh?    #
-#  Here's what the Apache server might look like   #
-#                                                  #
-#  This will assume an Express server is already   # 
-#                running on Port 3000              #
-#                                                  #
-####################################################
-
-<VirtualHost *:80>
-    ServerAdmin          admin@mycoolsite.com
-    ServerName           mycoolsite.com
```

Comment:
Our ServerName is not the name of what we want our site to be, but the name of the URL we're listening for.

If our server's IP address gets a request at Port 80 for 'mycoolsite.com', then we can say "Oh, that's us" and serve up Port 3000 (which we do below)